observatory_platform/telescopes/crossref_events.py

```diff
@@ -0,0 +1,701 @@
+        else:
+            batch_start = self.start_date.strftime("%Y-%m-%d")
+            batch_end = self.end_date.strftime("%Y-%m-%d")
+
+        if cursor:
+            return self.get_path(SubFolder.downloaded, f'{event_type}_{batch_start}_{batch_end}_cursor')
+        else:
+            return self.get_path(SubFolder.downloaded, f'{event_type}_{batch_start}_{batch_end}')
+
+    def subdir(self, sub_folder: SubFolder):
+        """
+        Path to subdirectory of a specific release for either downloaded/transformed files.
+
+        :param sub_folder:
+        :return:
+        """
+        date_str = self.start_date.strftime("%Y_%m_%d") + "-" + self.end_date.strftime("%Y_%m_%d")
+        return os.path.join(telescope_path(sub_folder, CrossrefEventsTelescope.DAG_ID), date_str)
+
+    def get_path(self, sub_folder: SubFolder, name: str) -> str:
+        """
+        Gets path to json file based on subfolder and name. Will also create the subfolder if it doesn't exist yet.
+
+        :param sub_folder: Name of the subfolder
+        :param name: File base name, without extension
+        :return: The file path.
+        """
+
+        release_subdir = self.subdir(sub_folder)
+        if not os.path.exists(release_subdir):
+            os.makedirs(release_subdir, exist_ok=True)
+
+        file_name = f"{name}.json"
+
+        path = os.path.join(release_subdir, file_name)
+        return path
+
+    @property
+    def blob_name(self) -> str:
+        """
+        Returns blob name that is used to determine path inside google cloud storage bucket
+
+        :return: The blob name
+        """
+        date_str = self.start_date.strftime("%Y_%m_%d") + "-" + self.end_date.strftime("%Y_%m_%d")
+
+        file_name = f"{CrossrefEventsTelescope.DAG_ID}_{date_str}.json"
+        blob_name = f'telescopes/{CrossrefEventsTelescope.DAG_ID}/{file_name}'
+
+        return blob_name
+
+
+def pull_release(ti: TaskInstance) -> CrossrefEventsRelease:
+    """
+    Pull a CrossrefEventsRelease instance with xcom.
+
+    :param ti: the Apache Airflow task instance.
+    :return: the CrossrefEventsRelease instance.
+    """
+
+    return ti.xcom_pull(key=CrossrefEventsTelescope.RELEASES_TOPIC_NAME,
+                        task_ids=CrossrefEventsTelescope.TASK_ID_DOWNLOAD, include_prior_dates=False)
+
+
+class CrossrefEventsTelescope:
+    """ A container for holding the constants and static functions for the crossref events telescope. """
+
+    DAG_ID = 'crossref_events'
+    DATASET_ID = 'crossref'
+    DESCRIPTION = 'The Crossref Metadata Plus dataset: ' \
+                  'https://www.crossref.org/services/metadata-retrieval/metadata-plus/'
+    RELEASES_TOPIC_NAME = "releases"
+    QUEUE = 'remote_queue'
+    MAX_PROCESSES = cpu_count()
+    MAX_CONNECTIONS = cpu_count()
+    MAX_RETRIES = 3
+    DOWNLOAD_MODE = 'parallel'  # Valid options: ['sequential', 'parallel']
+
+    MAILTO = 'aniek.roelofs@curtin.edu.au'
```

Comment:
It was intentional, since I was thinking we would replace it with a common email alias once this is created (see https://github.com/The-Academic-Observatory/observatory-platform/issues/78#issuecomment-691726153). 

I considered it similar to settings like download_mode, max_retries etc, however now that I think about it might be best to use an airflow connection/variable like you suggest, since we don't want others to use that e-mail when they clone the repository.